packages/nextjs/app/builders/0xaC1bBf2Aba61D6fC6fCbade72eCe3362de4724c3/page.tsx

```diff
@@ -0,0 +1,107 @@
+// app/builder/address/page.tsx
+
+const AVATAR_URL = "https://avatars.githubusercontent.com/u/583231?v=4";
+const WALLET_ADDRESS = "0xaC1bBf2Aba61D6fC6fCbade72eCe3362de4724c3";
+
+const SOCIALS = [
+  { name: "Twitter", url: "https://twitter.com/yourhandle", icon: "🐦" },
+  { name: "GitHub", url: "https://github.com/yourhandle", icon: "💻" },
+  { name: "LinkedIn", url: "https://linkedin.com/in/yourhandle", icon: "🔗" },
+];
+
+export default function Web3ProfilePage() {
+  return (
+    <main
+      style={{
+        minHeight: "100vh",
+        background: "linear-gradient(135deg, #e0e7ff 0%, #f0fdfa 100%)",
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        fontFamily: "Inter, Arial, sans-serif",
+      }}
+    >
+      <section
+        style={{
+          background: "#fff",
+          borderRadius: "18px",
+          boxShadow: "0 8px 32px rgba(0,0,0,0.12)",
+          padding: "40px 32px",
+          maxWidth: "400px",
+          width: "100%",
+          textAlign: "center",
+        }}
+      >
+        <img
+          src={AVATAR_URL}
+          alt="Avatar"
+          style={{
+            width: "100px",
+            height: "100px",
+            borderRadius: "50%",
+            objectFit: "cover",
+            boxShadow: "0 2px 8px rgba(0,0,0,0.10)",
+            marginBottom: "18px",
+          }}
+        />
```

Comment:
Consider using Next.js's `<Image>` component for better performance and built-in optimizations like lazy loading and responsive sizing.